Automattic/simplenote-ios
# Shortcuts update part 3: Added open new note intent and handler

### Fix
First shortcut to reimplement with siri kit is open new note intent.  This shortcut simply opens Simplenote and then creates a new note and opens that in the editor.  It doesn't take any parameters or have any actions, so is fairly simple.


### Test
1. Launch the app
2. go to the Shortcut app, create a new shortcut, add action, find Simplenote and select Open New Note.
3. Run the shortcut and confirm the app opens and creates a new note

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
